app/models/school_data_importer.rb

```diff
@@ -54,4 +55,11 @@ def schools_csv_url
   def date_string
     Time.zone.now.strftime("%Y%m%d")
   end
+
+  def generate_dfe_number(row)
+    [
+      row.fetch("LA (code)"),
+      row.fetch("EstablishmentNumber"),
```

Comment:
I think we only need the `EstablishmentNumber`, since we already have the `LA (code)` and this is derived from the two?

Reply:
I thought about that, but generating it on the fly would need us to do a join on the database to fetch the local authority, so may be slighty less performant.

Reply:
True. I guess I just find it a bit odd to duplicate the data. Ignore me if you feel that's appropriate.

Reply:
I agree with @erbridge, I think storing the establishment number and deriving the combined "number" in the UI makes more sense. It gives us more flexibility in terms of what we can do with the separate parts.